addons/resource/views/resource_calendar_views.xml

```diff
@@ -54,12 +54,11 @@
                         <group>
                             <field name="active" invisible="1"/>
                             <field name="company_id" groups="base.group_multi_company"/>
-                            <field name="hours_per_day" widget="float_time"/>
-                            <field name="tz" widget="timezone_mismatch" options="{'tz_offset_field': 'tz_offset', 'mismatch_title': 'Timezone Mismatch : This timezone is different from that of your browser.\nPlease, be mindful of this when setting the working hours or the time off.'}" />
-                            <field name="tz_offset" invisible="1"/>
+                            <field name="hours_per_day" widget="float_time" readonly="not flexible_hours"/>
+                            <field name="flexible_hours"/>
                         </group>
                     </group>
-                    <notebook>
+                    <notebook invisible="flexible_hours">
```

Comment:
What's the styling if there's an empty notebook?
If another module or a customisation adds new pages to the notebook, they'll be hidden as well

Reply:
I set the invisible condition on the page `working_hours` instead, thanks for the insight!